invenio_app_ils/circulation/config.py

```diff
@@ -315,3 +315,6 @@
         },
     )
 )
+
+# Feature Toggles
+CIRCULATION_SELF_CHECKOUT_ENABLED = SELF_CHECKOUT_ENABLED
```

Comment:
Could you clarify why is this assignment needed?

Reply:
Since, circulation is an extension it uses the circulation config and not the main config, so this is needed to get the value from the main config.

Reply:
I wouldn't fit this variable under circulation because we are not bringing this change to the circulation module. The prefixes like `CIRCULATION_` or `ILS_` are meant to hint from which module the variable is coming from